### DISCUSSION TITLE
Using the gamma band in EEG processing?

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [raphaelvallat/yasa#169] [Q&A] [answered]
In actual clinical practice, only up to 30Hz of the beta band is filtered. This algorithm even has gamma, but is there enough artifact in the brain waves?

**Comment by raphaelvallat** [comment, marked as answer]
I assume that you are referring to the [yasa.bandpower](https://raphaelvallat.com/yasa/build/html/generated/yasa.bandpower.html#yasa.bandpower function)? Note that you can disable the gamma band with:

```python
yasa.bandpower(..., bands=[(0.5, 4, 'Delta'), (4, 8, 'Theta'), (8, 12, 'Alpha'), (12, 16, 'Sigma'), (16, 30, 'Beta')])
```

It's up to you and your data whether you want to include the (low)-gamma band or not. I generally discard it and only include up to 30 or 35 Hz.
